dotnet/CrossLanguage/automation/Program.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using Pulumi.Automation;
+
+namespace CrossLanguage
```

Comment:
This one is just hanging on the `pulumi up` call and is a black box:

```
PS C:\git\pulumi-auto-examples\dotnet\CrossLanguage\automation> dotnet run                                                                                                                                                                                                                                                                             successfully initialized stack
setting up config...
config set
refreshing stack...
refresh complete
updating stack...
```

Reply:
This one is working when run normally from the pulumi project directory, but still is hanging when run from automation... I'm hoping the PR that fixes the delegate that allows us to pipe to console correctly will provide more insight about where it is hanging

Reply:
This works now I'm thinking it may have been a bug that was also resolved in that recent alpha publish because running it again didn't take any significant amount of time at any one point so I'm not sure where it was hanging before.